lib/elixir/lib/system.ex

```diff
@@ -65,8 +65,12 @@ defmodule System do
   @typedoc """
   The time unit to be passed to functions like `monotonic_time/1` and others.
 
-  The `:second`, `:millisecond`, `:microsecond` and `:nanosecond` time
-  units controls the return value of the functions that accept a time unit.
+  The `:day`, `:hour`, `:minute`, `:second`, `:millisecond`, `:microsecond`
+  and `:nanosecond` time units controls the return value of the functions
+  that accept a time unit.
+
+  Note that `:day`, `:hour`, and `:minute` where added in Elixir v1.11.4 and
```

Comment:
I wasn't sure if there was a good way to document this with `@since`, so I just made a note here. Version might need changing depending when this gets merged.